samples/image-create-and-upload-sample/image_create_and_upload_script.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+source config.sh
+ 
+function standardize_object_name() {
+  log "Formatting Object name $1"
+  local object_original_url=$1
+  local object_original_name=${object_original_url##*/}
+  local object_extension=""
+  local object_temp_name=""
+  if echo $object_original_name | grep -q -i rhcos; then
+    echo $object_original_name | grep 'qcow2.gz' >/dev/null
+    [ $? -ne 0 ] && error "Unsupported file format"
+    object_extension="ova.gz"
+    object_temp_name=${object_original_name%.*.*}
+  elif echo $object_original_name | grep -q -i rhel; then
+    echo $object_original_name | grep 'qcow2' >/dev/null
+    [ $? -ne 0 ] && error "Unsupported file format"
+    object_extension="ova.gz"
+    object_temp_name=${object_original_name%.*}
+  fi
+  OBJECT_NAME=`echo $object_temp_name | sed 's/\.//g'| sed 's/ppc64le//g'| sed 's/openstack//g' | sed 's/kvm//g' | sed -e 's/---*//g' | sed -e 's/_/-/g'| grep -o -E '[a-z]*-[0-9]{2}'`-$(date +%m%d%Y).${object_extension}
```

Comment:
Possible to combine the multiple sed patterns - https://www.cyberciti.biz/faq/howto-sed-substitute-find-replace-multiple-patterns/ ?

Also is the object name something like <name>-<xy>-<ddmmyyyy> ?
where `xy` is release version - 45, 46, 461 etc..

Reply:
Combined sed patterns
The object name will be in form (rhel/rhcos)-xy-mmddyyyy.ova.gz
xy will be release version like 46,45,83